horace_core/sqw/@sqw/calculate_qw_pixels2.m

```diff
@@ -129,10 +129,10 @@
 % into common coordinate system related to crystal (hkl or crystal
 % Cartesian depending on input)
 spec_to_rlu  = arrayfun(...
-    @(ex) calc_proj_matrix(ex,alatt, angdeg,n_matrix), ix_exper, 'UniformOutput', false);
+    @(ex) calc_proj_matrix(ex,alatt, angdeg,matrix_id), ix_exper, 'UniformOutput', false);
 
 % allocate common space for result. Improves performance.
-qw = zeros(4,numel(run_id));
+qw = zeros(4,numel(run_id)); % number of run_id is always equal to number of pixels processed
```

Comment:
following discussion last Thurs, replace numel(run_id) with size(idx,2) and remove the comment, as the easiest way to avoid having to note that the "run_id" variable has duplicates